src/graphics/engine/text.cpp

```diff
@@ -116,19 +117,26 @@ CText::~CText()
 
 bool CText::Create()
 {
+    CFontConfig fontConfig;
+    if (!fontConfig.Init())
+    {
+        GetLogger()->Warn("Error on parsing fonts config file: failed to open file\n");
+    }
     if (TTF_Init() != 0)
     {
         m_error = std::string("TTF_Init error: ") + std::string(TTF_GetError());
         return false;
     }
-
-    m_fonts[FONT_COLOBOT]        = MakeUnique<MultisizeFont>("fonts/dvu_sans.ttf");
-    m_fonts[FONT_COLOBOT_BOLD]   = MakeUnique<MultisizeFont>("fonts/dvu_sans_bold.ttf");
-    m_fonts[FONT_COLOBOT_ITALIC] = MakeUnique<MultisizeFont>("fonts/dvu_sans_italic.ttf");
-
-    m_fonts[FONT_COURIER]        = MakeUnique<MultisizeFont>("fonts/dvu_sans_mono.ttf");
-    m_fonts[FONT_COURIER_BOLD]   = MakeUnique<MultisizeFont>("fonts/dvu_sans_mono_bold.ttf");
-
+    
+    m_fonts[FONT_COMMON]        = MakeUnique<MultisizeFont>(fontConfig.GetFont(FONT_COMMON));
```

Comment:
Some ugly repetition again, but in this case I have no idea how to write it better as the values are bitmasks so there is no easy way to go through all of them.

Reply:
At first thought my idea was to get only primary font type (common, studio or satcom), and for every font type match regular, bold and italic variant. This way we would have 9 font types available in game. But I couldn't figure out how to do this without messing in text code.

Reply:
Untested pseudocode:
```C++
for(auto type : {FONT_COMMON, FONT_EDITOR, FONT_SATCOM})
{
    Load(type);
    Load(type|FONT_BOLD);
    Load(type|FONT_ITALIC);
}
```

You can check subtype when choosing filename with (font&FONT_BOLD)!=0 etc. Other types would be unused until we add something hat needs them.